dragonchain/broadcast_processor/broadcast_functions_utest.py

```diff
@@ -227,10 +227,11 @@ def test_set_record_for_block_sync_raises_when_not_accepting_level(self, mock_ge
             exceptions.NotAcceptingVerifications, broadcast_functions.set_receieved_verification_for_block_from_chain_sync, "block_id", 2, "chain_id"
         )
 
+    @patch("dragonchain.broadcast_processor.broadcast_functions.redis.lpush_sync")
     @patch("dragonchain.broadcast_processor.broadcast_functions.dragonnet_config.DRAGONNET_CONFIG", {"l2": {"nodesRequired": 3}})
     @patch("dragonchain.broadcast_processor.broadcast_functions.verifications_key", return_value="key")
     @patch("dragonchain.broadcast_processor.broadcast_functions.get_current_block_level_sync", return_value=2)
-    def test_set_record_for_block_sync_calls_redis_with_correct_params(self, mock_get_block_level, mock_key):
+    def test_set_record_for_block_sync_calls_redis_with_correct_params(self, mock_get_block_level, mock_key, mock_zadd):
```

Comment:
these should probably be mock_lpush, not zadd (x3 for this file)

Reply:
even better -> mock_sadd